src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,45 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static int buyReport = 0;
+    private static int supplyReport = 0;
 
     public void getStatistic(String fromFileName, String toFileName) {
-
+        String fileLineContent;
+        StringBuilder str = new StringBuilder();
```

Comment:
```suggestion
	static int buyReport = 0;
    	int supplyReport = 0;
        String fileLineContent;
        StringBuilder str = new StringBuilder();
```
тобі потрібно ці дві змінні оголосити ме